### PR TITLE
feat(numbertest1.py): add new file

### PR DESCRIPTION
**Pull request by ctrlbaltdelete**
holla

### CODE REVIEW

File: numbertest1.py [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+end_time = datetime.now()
+
+total_time = end_time - start_time
+print(total_time)
+
+
+
+
+
+
```
Comment (LesterKim) [inline numbertest1.py:35]: Please remove extra empty lines:
```suggestion
```

File: numbertest1.py [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+# def double(x):
+#   return 2 * x
+
+
+# numbers = list(range(1, 10))
+# total = reduce(lambda accumulator, number: accumulator + number, numbers, 0)
+# print(total == 45)
+
+# [double(number) for number in numbers]
```
Comment (LesterKim) [inline numbertest1.py:13]: Please remove comments:
```suggestion
 def double(x):
   return 2 * x


 numbers = list(range(1, 10))
 total = reduce(lambda accumulator, number: accumulator + number, numbers, 0)
 print(total == 45)

 [double(number) for number in numbers]
```

File: numbertest1.py (hunk 1 above)
Comment (ctrlbaltdelete) [reply numbertest1.py]: Fixed this

File: numbertest1.py (hunk 2 above)
Comment (ctrlbaltdelete) [reply numbertest1.py]: Fixed it.

**Pull request merged**